src/test/regress/multi_schedule

```diff
@@ -227,8 +227,10 @@ test: multi_citus_tools
 
 # ----------
 # multi_foreign_key tests foreign key push down on distributed tables
+# foreign_key_to_reference_table tests foreign constraints from distributed to reference tables
 # ----------
 test: multi_foreign_key
+test: foreign_key_to_reference_table
```

Comment:
The problem is that we use `table_fkey_cols` inside `table_fkeys_in_workers` and when we run the tests in parallel, because of some of the metadata table joins inside `table_fkey_cols` we get inconsistent outputs. 
Challenging thing to create this view is that we need to use `run_command_on_workers` and it needs to collect all of the results into 1 cell per worker. This is the reason why the view seems complicated. It gathers different column values in aggregates them into a json array.

This can be resolved by just not using `table_fkeys_in_workers` and using `\c`. However, this way, we have to set the `next_shard_id` and `search_path` after every check. I think this is an ugly solution. 

I understand that you want to run these two tests in parallel but currently we are already in a bad shape in terms of testing times. I think that we will soon be able to run travis tests so much quickly. I would love to keep the tests sequential instead of creating a test file full of `set citus.next_shard_id` and `search_path`.

Reply:
can't we run it in parallel with any other test?

Reply:
I run it in parallel with `multi_utilities`. Acceptable?